cmd/gossamer/configcmd.go

```diff
@@ -166,9 +166,10 @@ func createP2PService(fig *p2p.Config) *p2p.Service {
 func setBootstrapNodes(ctx *cli.Context, fig *p2p.Config) {
 	var urls []string
 
-	if ctx.GlobalIsSet(utils.BootnodesFlag.Name) {
+	if bnodes := ctx.GlobalString(utils.BootnodesFlag.Name); bnodes != "" {
 		urls = strings.Split(ctx.GlobalString(utils.BootnodesFlag.Name), ",")
```

Comment:
Consider using `bnodes` since you already have it declared

Reply:
They have to be declared separately, `urls []string` and `bnodes string`.

Reply:
oh I thought you were suggesting rather than declaring `urls` and `bnodes`, just to declare `bnodes` !